makank-website/src/app/new-post/new-post.component.ts

```diff
@@ -73,9 +73,66 @@ export class NewPostComponent implements OnInit {
         studentHousing: [ '', ''],
         pool: [ '', ''],
         garden: [ '', ''],
-        universities :[ '', '']
       },
     );
+
+
+    this.route.queryParams.subscribe((params:any) =>{
+      console.log(params.data)
+      if(params.data!=null){
+            this.editing=true
+            console.log("edit post with id = ",params.data);
+            this.editPost.postID=params.data
+            
+            // get post by id   >> request lma a3mlo ams7 al dummy data de
+      
+            // this.http.get<Post>("http://localhost:8080/posts/info/params.data").subscribe((data:any) =>{
```

Comment:
use the request to get the post from the database instead of the dummy data

Reply:
Waiting for database handling to use it